frontend/src/components/NavBar.js

```diff
@@ -141,6 +142,41 @@ class NavBar extends React.Component<{
         });
     }
 
+    // Fetch the image file represented by the bundle
+    fetchImg(bundleUuid) {
+        if (bundleUuid == null) return;
+        // Set defaults
+        let url = '/rest/bundles/' + bundleUuid + '/contents/blob/';
```

Comment:
Interesting method to render the image... could we simplify this by just adding `<img src=".../rest/bundles/.../contents/blob/" />` directly in the render function instead?

Reply:
it does not work if we directly use this method in the render function